v4/feature/portfolio/src/main/java/exchange/dydx/trading/feature/portfolio/components/positions/DydxPortfolioPositionItemView.kt

```diff
@@ -355,27 +356,28 @@ object DydxPortfolioPositionItemView {
                 .padding(0.dp),
             verticalArrangement = Arrangement.Bottom,
         ) {
-            /*
-            TODO: Only render the button if it is an isolated margin position
-             */
             Spacer(modifier = Modifier.weight(1.0f))
-            PlatformIconButton(
-                modifier = Modifier
-                    .width(32.dp)
-                    .height(32.dp),
-                action = {
-                    position.onAdjustMarginAction?.invoke()
-                },
-                padding = 0.dp,
-                shape = RoundedCornerShape(4.dp),
-                backgroundColor = ThemeColor.SemanticColor.layer_6,
-                borderColor = ThemeColor.SemanticColor.layer_7,
-            ) {
-                Icon(
-                    painter = painterResource(id = exchange.dydx.trading.common.R.drawable.ic_edit),
-                    contentDescription = "",
-                    tint = ThemeColor.SemanticColor.text_primary.color,
-                )
+            if (position.childSubaccountNumber != null) {
```

Comment:
can do `position.marginMode == isolated` for clarity. Also. I'm not sure but does subaccount 0 have childSubaccountNumber == 0? Guessing not, but haven't seen in abacus.

Reply:
Updated